extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/aggregation/bloom/BloomFilterBufferAggregator.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.bloom;
+
+import com.fasterxml.jackson.databind.util.ByteBufferBackedOutputStream;
+import org.apache.druid.io.ByteBufferInputStream;
+import org.apache.druid.query.ColumnSelectorPlus;
+import org.apache.druid.query.aggregation.BufferAggregator;
+import org.apache.druid.query.aggregation.bloom.types.BloomFilterAggregatorColumnSelectorStrategy;
+import org.apache.druid.query.monomorphicprocessing.RuntimeShapeInspector;
+import org.apache.hive.common.util.BloomKFilter;
+
+import java.nio.ByteBuffer;
+
+
+public class BloomFilterBufferAggregator implements BufferAggregator
+{
+  private ColumnSelectorPlus<BloomFilterAggregatorColumnSelectorStrategy> selectorPlus;
+  private int maxNumEntries;
+
+  public BloomFilterBufferAggregator(
+      ColumnSelectorPlus<BloomFilterAggregatorColumnSelectorStrategy> selectorPlus,
+      int maxNumEntries
+  )
+  {
+    this.selectorPlus = selectorPlus;
+    this.maxNumEntries = maxNumEntries;
+  }
+
+  @Override
+  public void init(ByteBuffer buf, int position)
+  {
+    final ByteBuffer mutationBuffer = buf.duplicate();
+    mutationBuffer.position(position);
+    BloomKFilter filter = new BloomKFilter(maxNumEntries);
+    ByteBufferBackedOutputStream wat = new ByteBufferBackedOutputStream(mutationBuffer);
```

Comment:
What does this variable name mean?

Reply:
Heh, oops, missed some clean up before opening the PR